crates/testing/src/test_runner.rs

```diff
@@ -411,29 +410,22 @@ where
             self.next_node_id += 1;
             tracing::debug!("launch node {}", i);
 
+            let all_nodes = config.known_nodes_with_stake.clone();
+            let da_nodes = config.known_da_nodes.clone();
+
             let memberships = Memberships {
-                quorum_membership: <TYPES as NodeType>::Membership::create_election(
-                    known_nodes_with_stake.clone(),
-                    known_nodes_with_stake.clone(),
+                quorum_membership: <TYPES as NodeType>::Membership::new(
+                    all_nodes.clone(),
+                    all_nodes.clone(),
                     Topic::Global,
+                    #[cfg(feature = "fixed-leader-election")]
                     config.fixed_leader_for_gpuvid,
                 ),
-                da_membership: <TYPES as NodeType>::Membership::create_election(
-                    known_nodes_with_stake.clone(),
-                    config.known_da_nodes.clone(),
+                da_membership: <TYPES as NodeType>::Membership::new(
+                    all_nodes,
+                    da_nodes,
```

Comment:
Eligible nodes to be leader == all nodes, but DA nodes are the only ones with voting power